LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-size-adjust-valid.html

```diff
@@ -2,22 +2,30 @@
 <html>
 <head>
 <meta charset="utf-8">
-<title>CSS Fonts Module Level 3: parsing font-size-adjust with valid values</title>
-<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-size-adjust-prop">
-<meta name="assert" content="font-size-adjust supports the full grammar 'none | [basis] <number>'.">
+<title>CSS Fonts Module Level 5: parsing font-size-adjust with valid values</title>
+<link rel="help" href="https://www.w3.org/TR/css-fonts-5/#font-size-adjust-prop">
+<meta name="assert" content="font-size-adjust supports the full grammar 'none | [metric]? [from-font | <number>]'.">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="/css/support/parsing-testcommon.js"></script>
 </head>
 <body>
 <script>
 test_valid_value('font-size-adjust', 'none');
+
 test_valid_value('font-size-adjust', '0.5');
 test_valid_value('font-size-adjust', 'ex-height 0.5', '0.5');  // default basis 'ex' omitted from serialization
 test_valid_value('font-size-adjust', 'cap-height 0.8');
 test_valid_value('font-size-adjust', 'ch-width 0.4');
 test_valid_value('font-size-adjust', 'ic-width 0.9');
 test_valid_value('font-size-adjust', 'ic-height 0.9');
+
+test_valid_value('font-size-adjust', 'from-font');
+test_valid_value('font-size-adjust', 'ex-height from-font', 'from-font');  // default basis 'ex' omitted from serialization
+test_valid_value('font-size-adjust', 'cap-height from-font');
+test_valid_value('font-size-adjust', 'ch-width from-font');
+test_valid_value('font-size-adjust', 'ic-width from-font');
+test_valid_value('font-size-adjust', 'ic-height from-font');
```

Comment:
would be nice to test some invalid combinations involving `from-font` as well.

Reply:
Would you say more? Do you mean a typo error, something like 'font-from'? Or else, extra value cases like `font-size-adjust: from-font 0.5` or `font-size-adjust: ex-height 0.5 from-font`?

Reply:
Extra value cases, like the ones you suggested, or `font-size-adjust: from-font from-font` etc. These are usually common parsing mistakes

Reply:
O.K. I added some invalid tests to imported/w3c/web-platform-tests/css/css-fonts/parsing/font-size-adjust-invalid.html

Reply:
Thanks! Please make sure to export them to WPT whenever you can, otherwise changes will be lost at the next import.

Reply:
Sure. Will do!